static/app/types/debugFiles.tsx

```diff
@@ -65,6 +65,30 @@ export type AppStoreConnectValidationData = {
   updateAlertMessage?: string;
 };
 
+export type AppStoreConnectStatusData = {
```

Comment:
@priscilawebdev this probably has a bulk of what you're interested in.

i went for 
```appstoreCredentialsValid: {is_valid: boolean} | {error: string; detail: string};``` because i'm thinking that it's a little easier to work around type-wise for you instead of say ```{ status: string; detail: string | undefined }```. happy to change it to something else though.

i'm also thinking that `detail` actually is unnecessary for us to return since you can probably just derive a string to be shown to the user given some error value. the error's supposed to be an enum anyways but it's a WIP i'm just not sure what its range of values look like so far. have to get some test data set up first. back to `detail` though, i would imagine not including it can cut down on payload sizes and it's probably not too compatible with i18n anyways.

thoughts?

Reply:
oh... maybe we can just assume that if the backend returns an error then it's invalid... so it would be something like `{error: string | null}`? otherwise, I think I would still prefer` { status: string; detail: string | undefined }`